enterprise_access/apps/bffs/mixins.py

```diff
@@ -23,6 +23,10 @@ class EnterpriseCustomerUserSubsidiesDataMixin(BFFContextDataMixin):
     Mixin to access enterprise customer user subsidies data from the context.
     """
 
+    @property
+    def enterprise_customer_uuid(self):
+        return self.context.enterprise_customer.get('uuid')
```

Comment:
[nit/inform] There is a helper property `self.context.enterprise_customer_uuid` to make accessing the enterprise UUID more straightforward.

Reply:
[nit/consideration] Given `enterprise_customer_uuid` is available for all handler classes, does it make sense to only add this property within `EnterpriseCustomerUserSubsidiesDataMixin` vs. putting it somewhere more commonly accessible across all handlers, if it's needed (e.g., would it also potentially make sense to always grab the enterprise uuid from `self.context` throughout the handlers so its more explicit where it's coming from)?

Reply:
Ah same here, I'll switch to self.context.enterprise_customer_uuid.